app/policies/group_policy.rb

```diff
@@ -48,6 +48,14 @@ def invite?
     user.is_admin? or user.owns(record) or member? && [Group::MEMBERSHIP[:open], Group::MEMBERSHIP[:invitation]].include?(record.membership)
   end
 
+  def add?
+    user.is_admin?
+  end
+
+  def remove?
+    user.is_admin?
+  end
+
```

Comment:
Similar thing to the controller, I'd prefer add?/remove? to be before join? etc. rather than between invite?/reject?.
I also think the names should be add_user?/remove_user?, because groups can also have contests and problem sets added and removed.